cicd/pipelines/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -63,13 +63,21 @@ include::modules/op-creating-webhooks.adoc[leveloffset=+1]
 
 include::modules/op-triggering-a-pipelinerun.adoc[leveloffset=+1]
 
+include::modules/op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="_additional-resources_op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects"]
+.Additional resources
```

Comment:
Honestly I'd get another pair of eyes from the core team to check this - I _think_ it needs to be a H2 instead of a .heading and the ID I believe should be per the assembly, not the module name before it? @rolfedh any idea?

Reply:
tbh I think it would be better to just add this to the existing additional resources section: 
![image](https://user-images.githubusercontent.com/40172997/172629977-36167569-9f59-4f1c-aa9a-6887aa7e5f6f.png)

Reply:
+1 I agree with @abrennan89' recommendation. Disregard my previous comment. 

Normally we would have moved the `.Additional resources` section like shown, because this is the recommended workaround for the "no xrefs in modules" issue. However, in this case, it looks wrong having two additional resources sections stacked on top of each other. I didn't notice that at the time I made the recommendation.

Reply:
I support @abrennan89's recommendation. Let's eliminate the module-level `.Additional resources` and move the xref to the following `== Additional resources` section.